renderer/src/App.tsx

```diff
@@ -30,6 +31,7 @@ function Home (): JSX.Element {
   return (
     <div style={{ marginTop: '2em' }}>
       <h2>Welcome to Filecoin Station</h2>
+      <JobCounter />
```

Comment:
To match the visible UI, shall we rename this component either to `TotalJobsCompleted` or `JobStats`?